Task3/EEL_task_3/src/main/java/com/epam/esm/Application.java

```diff
@@ -0,0 +1,14 @@
+package com.epam.esm;
+
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+
+@SpringBootApplication
+@EnableWebMvc
```

Comment:
Why do we need this annotation?

Reply:
We need this annotation to configure the entry point: to configure SpringBoot and to enable ComponentScan on the package.